client/post-editor/editor-sidebar/feedback-view.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FeedbackSidebarHeader from './feedback-header';
+import FeedbackRequestForm from './feedback-request-form';
+import FeedbackList from './feedback-list';
+import SidebarFooter from 'layout/sidebar/footer';
+
+// TODO: Find a clearer word than "share" for the owner of feedback and use it in subcomponent and CSS classes
+export class FeedbackView extends PureComponent {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		close: PropTypes.func.isRequired,
+		sharedLinks: PropTypes.array.isRequired,
+	};
+
+	state = { openFeedbackCount: 0 };
+
+	onToggleFeedback = isOpen => {
+		const { openFeedbackCount } = this.state;
+
+		this.setState( {
+			openFeedbackCount: openFeedbackCount + ( isOpen ? 1 : -1 ),
```

Comment:
I thought the code could be made clearer, so I attempted to do that. But I did end up adding a comment on the initial state assignment. What do you think of the updates?